src/TimeGrid.js

```diff
@@ -188,15 +193,17 @@ export default class TimeGrid extends Component {
         {this.renderHeader(range, allDayEvents, width, resources)}
 
         <div ref="content" className="rbc-time-content">
-          <div ref="timeIndicator" className="rbc-current-time-indicator" />
-
-          <TimeColumn
-            {...this.props}
-            showLabels
-            style={{ width }}
-            ref={gutterRef}
-            className="rbc-time-gutter"
-          />
+          {!hideTimeIndicator && <div ref="timeIndicator" className="rbc-current-time-indicator" />}
```

Comment:
If we aren't showing the indicator we should also not be calculating it's position on an interval. Can you also disable that when this is true?